airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -819,7 +819,7 @@ def process_pod_deletion(self, pod: k8s.V1Pod, *, reraise=True):
                     self.log.info("Deleting pod: %s", pod.metadata.name)
                     self.pod_manager.delete_pod(pod)
                 elif should_delete_pod and istio_enabled:
-                    self.log.info("Deleting istio-proxy sidecar inside %s: ", pod.metadata.name)
+                    self.log.info("Deleting istio-proxy sidecar inside %s:", pod.metadata.name)
```

Comment:
```suggestion
                    self.log.info("Deleting istio-proxy sidecar inside: %s", pod.metadata.name)
```

This seems to better match the format in other messages